src/CMakeLists.txt

```diff
@@ -7,20 +7,21 @@ ExternalProject_Add(UFS_UTILS
   CMAKE_ARGS "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}"
   )
 
-
-set(CCPP_SUITES "FV3_CPT_v0,FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_regional,FV3_GSD_SAR,FV3_GSD_v0,FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_RRFS_v1beta")
+if(NOT CCPP_SUITES)
+  set(CCPP_SUITES "FV3_CPT_v0,FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_regional,FV3_GSD_SAR,FV3_GSD_v0,FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_RRFS_v1beta")
```

Comment:
Should we add RRFS_v1alpha here, or is that not in the relevant hash of ufs-weather-model yet?

Reply:
Good catch, I didn't think about that change here...the change I made bringing over RRFS_v1alpha to develop was in regional_workflow, but as it turns out the necessary suite definition file is not yet in ufs-weather-model's develop, so it will not work anyway. So I would say we don't need to include it for now, and should add it when/if the SDF gets added to develop.